### ISSUE TITLE
A11y_AzureDataStudioStayHealthy_Notebook_Toolbar_ColorContrast: The separator color contrasts is below 3:1

### ISSUE THREAD

**Issue by anthonydresser** [anthonydresser/testissues#3410]
["Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:**-A11y_AzureDataStudioStayHealthy;-A11yTCS;-A11yWCAG2.1;-Benchmark-DesktopApp;-MAS1.4.11;-SQL Azure Data Studio;-FTP;-ColorContrast;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.20.0-insider (system setup)
Commit: fd29e35
Date: 2020-07-02T05:40:02.292Z
VS Code: 1.46.0
Electron: 7.3.2
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.18363

**Additional Details:**
MAS References:MAS1.4.11
Tool: Accessibility insights for desktop

**Repro Steps:**
1.Launch Azure Data Studio Insiders application.
2. Connect to server.
3. Right-click connection and click New Notebook
4. Click on cell and select code cell navigate to toggle more and open it .
5. Launch Accessibility insights for desktop and select third tab contrast.

**Actual:**
When the color contrast of the separator under toggle more button is checked the ratio is 1.557:1

**Expected:**
The color contrast of the separator should be 3:1 or greater

**User Impact:**
Low vision users will face difficulty in viewing the separate sections in the lsit .

**Attachment link for Reference:**
